python/packages/core/agent_framework/_workflows/_handoff.py

```diff
@@ -1236,6 +1283,92 @@ async def check_termination(conv: list[ChatMessage]) -> bool:
         self._termination_condition = condition
         return self
 
+    def with_interaction_mode(
+        self,
+        interaction_mode: Literal["human_in_loop", "autonomous"] | None = "human_in_loop",
```

Comment:
what would `None` mean here? I think just the literal and the default is enough. Calling this with None doesn't make any sense.